src/main/java/org/broadinstitute/hellbender/tools/gvs/common/WeightedSplitIntervals.java

```diff
@@ -41,7 +41,6 @@
         programGroup = IntervalsManipulationProgramGroup.class
 )
 @DocumentedFeature
-@ExperimentalFeature
```

Comment:
experimental no more!

Reply:
Talk to @lbergelson  -- the Experimental tag in GATK means something specific about support.  Many significant tools (reblocking I believe) are in this category because there is not full frontline support for them.

Reply:
OK thanks... the team had expressed some concern about the seemingly alarming "experimental" warning message on startup in what now feels like a core part of a production(ish) pipeline.

Reply:
I agree with that, and I think it's how this came up the last time :D  It was something akin to Gmail being in Beta.  But Louis or @droazen can give us the scoop